package.json

```diff
@@ -355,7 +355,7 @@
   "engines": {
     "node": "0.10 || 0.12 || 4 || 5"
   },
-  "version": "0.13.21",
+  "version": "0.13.22",
```

Comment:
Done.  The problem now is that I'll have nothing to reference in the browserify change